Pi3BoardPkg/Pi3BoardPkg.fdf

```diff
@@ -144,8 +144,10 @@ READ_LOCK_STATUS   = TRUE
   #
   # MMC/SD
   #
-  INF Pi2BoardPkg/Drivers/MmcDxe/MmcDxe.inf
-  INF Pi2BoardPkg/Drivers/SdHostDxe/SdHostDxe.inf
+  #INF Pi3BoardPkg/Drivers/MmcDxe/MmcDxe.inf
```

Comment:
Do we expect anyone to uncomment these lines in the future? If not, please remove.